### PR TITLE
docs(auto-rules): add docs for uploading/downloading automated rules

### PR DESCRIPTION
**Pull request by tthvo**
Related to https://github.com/cryostatio/cryostat-web/issues/447
Related to #45 
Depends on https://github.com/cryostatio/cryostatio.github.io/issues/54

Add documentation for uploading/downloading automatic rules.

Also, updated a screenshot of the main view of Automated Rule and Events as we now have an upload icon button.

### CODE REVIEW

**Comment by andrewazores** [comment]
> Since we have a new upload icon button in other views, I wonder if we should also update those screenshots too?

Good call. Want to file some issues for that?

File: guides/_subsections/upload-an-automated-rule.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+## [Upload an Automated Rule](#upload-an-automated-rule)
+
+Instead of stepping through the form every time, you can also edit your Automated Rule definition files (JSON format) locally and quickly upload them.
+
+Below is an example of a rule file content:
+
+{% highlight bash %}
```
Comment (andrewazores) [inline guides/_subsections/upload-an-automated-rule.md:7]: Is there a `json` highlight type too? Or is `bash` really the best fit? Even `javascript` might be suitable.

File: guides/_subsections/download-an-automated-rule.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Automated Rules can easily be downloaded for viewing and editting.
+
+
+<ol>
+  <li>
+      {% include howto_step.html
+        summary="Open action menu for a selected rule"
+        image-name="download-an-automated-rule-1.png"
+        caption="Click on kebab icon at the end of the selected rule row."
```
[the remark this answers is not in the document]
Comment (tthvo) [reply guides/_subsections/download-an-automated-rule.md:11]: oh yeh I was not sure how to call it but let me find and replace it with the "three dot" term

**Comment by tthvo** [comment]
> > Since we have a new upload icon button in other views, I wonder if we should also update those screenshots too?
> 
> ~Good call. Want to file some issues for that?~ I think that can fall under #45

Oh yeh I can update screenshots for upload icons and reference that issue here.

**Comment by tthvo** [comment]
I updated screenshots for views with new upload icon (i.e. Automated Rules and Events). 

Since there Archives view is not updated yet with its own view and section, I am not sure whether I should do the updates too or will be done later in #45 ?

**Review by andrewazores — APPROVED** [review]

**Pull request merged**